recover_affected_workloads.md

```diff
@@ -21,12 +21,17 @@ For example, in the case where the node fails to update its status to the
 masters the node condition will be set to `type:Ready status:Unknown`.
 
 In order to gain the workload back, reschedule it and start it all over, the
-control plane needs to know that the running node is gone and the only way to achieve
-that is by deleting the node object, which will trigger [scheduling the pods for
-deletion][2]
+control plane needs to know that the running node is gone and there are two ways to do it:
```

Comment:
```suggestion
control plane needs to know that the running node is gone and there are two ways for Non-graceful node shutdown handling[2]:
```

[2] https://kubernetes.io/docs/concepts/cluster-administration/node-shutdown/#non-graceful-node-shutdown

Reply:
this suggested change does not make sense, "non-graceful node shutdown" is the name of the feature which uses the out of service taint for dealing with nodes which are shut down but the control plane not knowing it yet